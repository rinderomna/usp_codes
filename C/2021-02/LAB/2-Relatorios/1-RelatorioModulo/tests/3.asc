100
-95931 -94173 -89706 -88820 -88577 -86974 -79378 -73977 -71457 -67742 -67445 -66474 -64006 -62799 -59418 -54914 -53408 -51259 -50407 -50159 -46897 -45040 -44183 -42297 -41372 -38415 -38206 -37674 -35557 -31337 -29246 -25011 -22993 -22656 -22066 -17192 -15711 -12326 -11687 -10726 -10262 -9131 -8529 -8299 -7187 -6527 -5051 -3549 -1489 2876 2959 4291 6149 6548 6586 8439 10183 12616 13253 14424 23938 24256 24449 25929 27155 29517 32945 33591 34929 38914 39308 42060 50423 51721 53656 54181 54309 56266 56950 58343 59652 64584 67320 68012 71841 71988 77175 77747 78936 78998 79426 79744 82926 86541 86823 87524 89954 93421 97273 98239
