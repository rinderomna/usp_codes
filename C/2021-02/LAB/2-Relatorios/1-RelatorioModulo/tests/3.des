100
98239 97273 93421 89954 87524 86823 86541 82926 79744 79426 78998 78936 77747 77175 71988 71841 68012 67320 64584 59652 58343 56950 56266 54309 54181 53656 51721 50423 42060 39308 38914 34929 33591 32945 29517 27155 25929 24449 24256 23938 14424 13253 12616 10183 8439 6586 6548 6149 4291 2959 2876 -1489 -3549 -5051 -6527 -7187 -8299 -8529 -9131 -10262 -10726 -11687 -12326 -15711 -17192 -22066 -22656 -22993 -25011 -29246 -31337 -35557 -37674 -38206 -38415 -41372 -42297 -44183 -45040 -46897 -50159 -50407 -51259 -53408 -54914 -59418 -62799 -64006 -66474 -67445 -67742 -71457 -73977 -79378 -86974 -88577 -88820 -89706 -94173 -95931
