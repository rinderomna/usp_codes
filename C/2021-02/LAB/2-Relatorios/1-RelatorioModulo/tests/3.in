100
-4405 69896 -77327 -7487 -77230 14723 -67831 59624 -89370 46403 49828 -40820 73409 -97673 -11095 20791 64503 85123 88111 -29662 -57827 -94175 21882 -3487 77643 -66542 15146 -98518 -84797 50744 3107 5107 -40138 -42161 98469 -22985 -5008 -40884 -16223 2166 7398 -8987 27282 -41249 22353 89308 -65484 70144 -24741 -73420 -783 49696 -79847 -5991 13062 -49223 -48842 -6622 -44127 -33973 90334 -15845 -95731 82475 -52244 -73383 -72660 24767 29108 98865 45098 87478 73372 1131 -74098 -2659 -89452 97321 91020 41986 95036 7539 20172 20349 -8284 -59179 30983 -13194 74073 30647 -6025 -95589 -35759 -90626 -89207 11176 34923 36122 -72810 87194
