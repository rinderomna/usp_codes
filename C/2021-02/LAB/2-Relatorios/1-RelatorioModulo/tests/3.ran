100
-10726 27155 32945 -22993 -89706 -86974 -10262 -50407 -29246 -50159 8439 51721 77175 -6527 -67742 24449 86823 -5051 78936 -62799 -59418 -46897 -8299 10183 6149 33591 56266 86541 53656 -66474 93421 79426 12616 -1489 77747 38914 -35557 24256 -42297 -15711 -79378 2876 -12326 6548 -8529 82926 -67445 -17192 -73977 71988 -88577 -11687 -71457 67320 -38206 25929 -94173 4291 -31337 -41372 58343 23938 68012 -51259 97273 -3549 54309 -64006 -44183 39308 -53408 -54914 -38415 56950 29517 79744 2959 71841 89954 6586 87524 78998 59652 34929 54181 -22656 -45040 50423 -95931 13253 42060 -7187 -22066 -25011 -88820 -37674 64584 98239 14424 -9131
