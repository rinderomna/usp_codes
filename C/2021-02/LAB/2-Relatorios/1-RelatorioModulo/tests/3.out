-98518 -97673 -95731 -95589 -94175 -90626 -89452 -89370 -89207 -84797 -79847 -77327 -77230 -74098 -73420 -73383 -72810 -72660 -67831 -66542 -65484 -59179 -57827 -52244 -49223 -48842 -44127 -42161 -41249 -40884 -40820 -40138 -35759 -33973 -29662 -24741 -22985 -16223 -15845 -13194 -11095 -8987 -8284 -7487 -6622 -6025 -5991 -5008 -4405 -3487 -2659 -783 1131 2166 3107 5107 7398 7539 11176 13062 14723 15146 20172 20349 20791 21882 22353 24767 27282 29108 30647 30983 34923 36122 41986 45098 46403 49696 49828 50744 59624 64503 69896 70144 73372 73409 74073 77643 82475 85123 87194 87478 88111 89308 90334 91020 95036 97321 98469 98865
